libs/providers/launchdarkly-client/src/lib/translate-context.spec.ts

```diff
@@ -0,0 +1,230 @@
+import translateContext from "./translate-context";
```

Comment:
Same as [here](https://github.com/open-feature/js-sdk-contrib/pull/343/files#r1184951465) if this is lifted from another repo.